Measures/AddingIdealAirLoads/measure.xml

```diff
@@ -63,6 +63,20 @@
       <required>true</required>
       <model_dependent>false</model_dependent>
     </argument>
+    <argument>
+      <name>hvacSchedFeiertag</name>
```

Comment:
Similar comment as above. 
I think we do not use this measure anymore, Should we just remove it?

Reply:
The name is adressed in a later commit (branch rename-measure-arguments). The measure is not added to the workflow by default. As far as I can tell we can safely remove the measure. (Added to tasks)